www/include/eventLogs/viewLog.php

```diff
@@ -130,20 +158,22 @@
 }
 
 $defaultServicegroups = array();
-if (isset($_GET['svcg'])) {
-    $svcg = explode(",", $_GET['svcg']);
+if (isset($getInputs['svcg'])) {
+    $svcg = explode(",", $getInputs['svcg']);
+    $svcGroupsIds = filteredArrayId($svcg);
     $serviceGrpObj = new CentreonServicegroups($pearDB);
-    $serviceGrpArray = $serviceGrpObj->getServicesGroups($svcg);
+    $serviceGrpArray = $serviceGrpObj->getServicesGroups($svcGroupsIds);
```

Comment:
The verification of ids should not be done here but directly in the getServicesGroups method to get a wider scope of application.